lax/lichens/sciencerun0.py

```diff
@@ -873,3 +875,35 @@ def _process(self, df):
                                      (13.719 * np.exp(-df['s2'] / 55.511) + 3.014)) &
                                     (df['s2'] <= 300))))
         return df
+
+
+class SingleElectronS2s(Lichen):  # noqa
+    """To classify S1s from single electron S2s.
+    Information: xenon:xenon1t:analysis:subgroup:cuts:meetings:20180126:se_classification
+    Contact: Fei Gao <feigao.ge@gmail.com>
+    """
+
+    version = 4
+
+    def _process(self, df):
+
+        # Random forest classifier
+        forest_filename = os.path.join(DATA_DIR, 'XENON1T_random_forest_peak_classifier_01262018.pkl')
+        forest_load = pickle.load(open(forest_filename, 'rb'))  # noqa
+
+        # Gradient Boosted Decesion Tree classifier
+        gbdt_filename = os.path.join(DATA_DIR, 'XENON1T_gradient_bdt_peak_classifier_01262018.pkl')
+        gbdt_load = pickle.load(open(gbdt_filename, 'rb'))  # noqa
```

Comment:
Do you just get pickle security warnings?

Reply:
Yeah, only saw those pickle warning. There is sth called __class__ not ... We discussed this in the [Katrina's SE PR too](https://github.com/XENON1T/hax/pull/203).